tools/coqdoc/main.ml

```diff
@@ -292,16 +292,16 @@ let parse () =
         Cdglobals.externals := false; parse_rec rem
     | ("--external" | "-external") :: u :: logicalpath :: rem ->
         Index.add_external_library logicalpath u; parse_rec rem
-    | ("--coqlib" | "-coqlib") :: u :: rem ->
-        Cdglobals.coqlib := u; parse_rec rem
-    | ("--coqlib" | "-coqlib") :: [] ->
+    | ("--coqlib_url" | "-coqlib_url") :: u :: rem ->
+        Cdglobals.coqlib_url := u; parse_rec rem
+    | ("--coqlib_url" | "-coqlib_url") :: [] ->
         usage ()
     | ("--boot" | "-boot") :: rem ->
         (* XXX: This is useless it seems *)
         parse_rec rem
-    | ("--coqlib_path" | "-coqlib_path") :: d :: rem ->
-        Cdglobals.coqlib_path := d; parse_rec rem
-    | ("--coqlib_path" | "-coqlib_path") :: [] ->
+    | ("--coqlib" | "-coqlib") :: d :: rem ->
+        Boot.Env.set_coqlib d; parse_rec rem
```

Comment:
Maybe you could check here if the argument has a URL format and print an informative error message about the renaming of the option.

Reply:
Thanks for the suggestion, done.